classes/ActionScheduler_WPCLI_QueueRunner.php

```diff
@@ -0,0 +1,220 @@
+<?php
+
+/**
+ * WP CLI Queue runner.
+ *
+ * This class can only be called from within a WP CLI instance.
+ */
+class ActionScheduler_WPCLI_QueueRunner extends ActionScheduler_Abstract_QueueRunner {
+
+	/** @var array */
+	protected $actions;
+
+	/** @var  ActionScheduler_ActionClaim */
+	protected $claim;
+
+	/** @var string */
+	protected $id;
```

Comment:
Renaming this `$claim_id` would help disambiguate it. I thought for a second the _runner_ itself might have been getting an ID rather than this value being used for a batch's claim ID.

Reply:
Actually on further investigation, the `$id` property seems unnecessary. It stores `$this->claim->get_id()`, so `$this->claim->get_id()` could be called directly instead on the one occasion where `$this->id` is used.